entx/template/event_hooks.tmpl

```diff
@@ -163,6 +163,9 @@
 								return nil, fmt.Errorf("failed to load object to get values for event, err %w", err)
 							}
 
+							// Satisfies the go error: dbObj declared and not used
```

Comment:
It does, but the better answer seems to be not to create dbObj if len($node.Fields) is 0, versus faking an access.